python/python/res/job_queue/job_manager.py

```diff
@@ -122,12 +123,17 @@ def __init__(self, module_file="jobs.py", json_file="jobs.json", error_url=None,
         self.max_runtime = 0  # This option is currently sleeping
         self.short_sleep = 2  # Sleep between status checks
         self.node = socket.gethostname()
-
         pw_entry = pwd.getpwuid(os.getuid())
         self.user = pw_entry.pw_name
+        v = EclVersion()
         logged_fields= {"status": "init",
                         "python_sys_path": sys.path,
                         "pythonpath": os.environ.get('PYTHONPATH', ''),
+                        "ecl_major_version": v.major,
+                        "ecl_minor_version": v.minor,
+                        "ecl_micro_version": v.micro,
+                        "ecl_version": v.versionString(),
```

Comment:
Yeah, I considered it, and I am still uncertain. Maybe we would like to have easy access to see who runs e.g. 2.1.$whatever, or how many runs 2.$whatever. As long as the v.versionString() format remains locked it is of course enough for that, but it feels ugly to lock that format. On the other hand, it is a fricking version string, why would we every want to change its format?